JeongJaeHeon/.gitignore

```
@@ -397,4 +397,7 @@ zsdoc/data
 /tests/_output/*
 !/tests/_output/.gitkeep
 
-# End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,vim,webstorm,macos,linux,windows,zsh,node
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,vim,webstorm,macos,linux,windows,zsh,node
+
+# schema.sql
+schema.sql
```

```suggestion
db/schema.sql
```

이미 github에 push 된 경우 `.gitignore`에 추가를 하여도 해당 파일이 적용이 되지 않습니다.

`git rm --cached "file name"` 명령어를 통해 해당 파일에 대한 캐시를 삭제 후 다시 push 해주세요!